public/js/components/RightSidebar.js

```diff
@@ -80,6 +80,11 @@ const RightSidebar = React.createClass({
 
   setupKeyboardShortcuts() {
     const { keyShortcuts } = this.props;
+    if (this.keyShortcutsEnabled) {
+      return;
+    }
+
+    this.keyShortcutsEnabled = true;
```

Comment:
The issue here is a frustrating lifecycle issue with react. The child event is first and so keyShortcuts is not setup yet.